src/sentry/static/sentry/app/views/performance/compare/utils.tsx

```diff
@@ -318,17 +323,62 @@ function createChildPairs({
   };
 }
 
+function jaroSimilarity(thisString: string, otherString: string): number {
+  // based on https://winkjs.org/wink-distance/string-jaro-winkler.js.html
+  // and https://en.wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance
+  if (thisString === otherString) {
+    return 1;
+  }
+
+  let jaroDistance: number = jaro(thisString, otherString).distance;
+
+  const scalingFactor = 0.1;
```

Comment:
Can we add a comment about this? I’d imagine we would want to change this in the future, so it’s nice to have context. Same with boost context below.

Reply:
Fixed in https://github.com/getsentry/sentry/pull/20224/commits/4005be21e1333ff9c22408a57261da8ecf7c7e55 

This is my best understanding of these parameters.